docs/pages/web/spinner.js

```diff
@@ -3,6 +3,12 @@ import { type Node } from 'react';
 import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
 import delay from '../../examples/spinner/delay.js';
 import docgen, { type DocGen } from '../../docs-components/docgen.js';
+import doLocation from '../../examples/spinner/doLocation.js';
```

Comment:
Overall hideControls in all Don't examples

Reply:
ah thanks, good catch